mariadb-image/root/usr/local/bin/entrypoint.sh

```diff
@@ -93,6 +94,27 @@ prepare() {
     chmod 600 $config $command
 }
 
+# Read MYSQLD_CONFIG envvar and write all settings separated by ';' character.
+# E.g. MYSQLD_CONFIG="skip-networking;max_allowed_packet=16M" will produce:
+#
+# ```
+# [mysqld]
+# skip-networking
+# max_allowed_packet=16M
+# ```
+prepare_mysqld_config() {
```

Comment:
Minor comment: I would merge both `prepare()` and `prepare_mysql_config()`, as they both prepare things and the first one is really small...

Reply:
I'd prefer to keep them split, since `prepare` creates the user `.my.cnf` whereas `prepare_mysqld_config` sets up the `/etc/my.cnf.d` mysqld settings.